docs/site/Serving-static-files.md

```diff
@@ -0,0 +1,64 @@
+---
+lang: en
+title: 'Serving static files in LoopBack 4'
+keywords: LoopBack
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Serving-static-files.html
+---
+
+One of the basic requirements of a web app is the ability to serve static files.
+Serving static files from a LoopBack is very simple - just call the
+`app.static(path, rootDir[, options])` method. The variables in the API are
+explained below.
+
+- `app`: An instance of a LoopBack application.
+- `path`: The path where the static assets are to be served from. Refer to
+  [path examples](https://expressjs.com/en/4x/api.html#path-examples) in the
+  Express docs for possible values.
+- `rootDir`: The directory where the static assets are located on the file
+  system.
+- `options`: An otional object for configuring the underlying
+  [express.static](https://expressjs.com/en/4x/api.html#express.static).
+
+Here is an example of configuring an app to serve the static assets from a
+directory named `public` at `/`.
+
+{% include code-caption.html content="src/application.ts" %}
+
+```ts
+export class TodoListApplication extends BootMixin(
+  ServiceMixin(RepositoryMixin(RestApplication)),
+) {
+  constructor(options: ApplicationConfig = {}) {
+    super(options);
+
+    // ...
+
+    this.static('/', path.join(__dirname, '../../public'));
```

Comment:
trying to verify your instruction.  Is it `options.path.join...` instead of `path.join`?  Because when i copied and pasted this line, it says "cannot find path".

Reply:
@dhmlau `path` should be a node module. The sample code could import it to fix your error.

Reply:
Added `path` dependency for clarity.